main.nf

```diff
@@ -18,10 +18,26 @@ nextflow.enable.dsl = 2
 
 WorkflowMain.initialise(workflow, params, log)
 
-include {METAGENOMICS_SR} from './workflows/metagenomics-sr'
+if (params.platform == 'illumina') {
```

Comment:
I am not sure if the inclusion needs to be conditional, are there examples of this in other places?